### PR TITLE
Update default ubuntu image to Jetstream 20.04

### PR DESCRIPTION
**Pull request by PermissionError**
See https://issues.openmrs.org/projects/ITSM/issues/ITSM-4277

### CODE REVIEW

File: global-variables.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 variable "image" {
-  default = "87e08a17-eae2-4ce4-9051-c561d9a54bde"
+  default = "64ad3b27-4d95-4c12-9c7f-1a85a73fdeff"
```
Comment (HerbertYiga) [inline global-variables.tf:2]: @PermissionError Thanks

Comment (HerbertYiga) [reply global-variables.tf:2]: @PermissionError  you only missed a ticket Id on your ticket description

Comment (PermissionError) [reply global-variables.tf:2]: @HerbertYiga Thanks for the reminder, I have amended the commit message with the Jira ticket.

**Comment by cintiadr** [comment]
I'm really sorry for the delay, I haven't had the energy or headspace to help openmrs recently. 

So I'm going to say that we _probably_ don't want to change the global value directly before testing a couple of machines. I think it's just a matter of passing another variable to the the module of the machine you want to start testing. 

And you will need to have terraform creds to be able to run this. 

Can you talk to Burke? I need you enrolled to jetstream. Otherwise you won't be able to run terraform commands. 
https://github.com/openmrs/openmrs-contrib-itsmresources/wiki/Provider-Jetstream

**Comment by PermissionError** [comment]
Thank you very much Cintia. No need to apologies, there's nothing wrong with being busy with life! I've started the process for obtaining terraform creds, thank for the pointers again :)

**Comment by PermissionError** [comment]
I'm going to close this PR and open a new one with a branch from this repo instead of my fork. Since work is also being done on ITSM-4285, the git changes will be messy if I keep them on my fork.

**Pull request closed without merging**